src/components/charts/BarChart.tsx

```diff
@@ -112,11 +113,18 @@ const BarChart = () => {
       azureSecondary.push(parseInt(item.qtsDCSessionsCount));
     });
 
+    const maxValue = Math.max(...azurePrimary, ...azureSecondary);
```

Comment:
keep constant declarations at the top of the file, rather than in middle of file.